.github/actions/setup-dev/action.yml

```diff
@@ -0,0 +1,41 @@
+name: Set up development
+description: Install required dependencies
+runs:
+  using: "composite"
+  steps:
+    - name: Install terraform
+      uses: hashicorp/setup-terraform@v2
+      with:
+        terraform_wrapper: false
+
+    - uses: actions/cache@v3
+      id: npm-cache
+      with:
+        path: ~/.npm
+        key: ${{ runner.os }}-build-${{ env.cache-name }}
+
+    - name: Install npm dependencies
+      shell: bash
+      run: npm install -g serverless serverless-scaleway-functions
```

Comment:
Can we split out the `npm`-related stuff into a separate action? It's only necessary for the SF integration tests I think.

Reply:
Excellent, thanks! Same goes with secrets :face_with_spiral_eyes: I was mounting them but not using them.